src/components/globe/globe-component.jsx

```diff
@@ -11,13 +11,15 @@ const GlobeComponent = ({ sceneId, sceneSettings, onLoad, children, loadElement
 
   const handleOnLoad = (map, view) => {
     setMap(map);
+    map.ground.surfaceColor = '#0A212E';
     setView(view);
     setLoaded(true);
-    onLoad && onLoad(map, view);
+    onLoad && onLoad(map, view);  
   }
 
   return (
     <WebScene
+      style={{ backgroundColor: '#0A212E' }}
```

Comment:
I'm not sure, could you explain what this `backgroundColor` does in this case? Thanks :)

Reply:
I set the background color to prevent the user from seeing a naked globe :)

Reply:
Talking about seeing a globe "naked" is there any way we can't avoid showing the satellite basemap when we land? I remember having some conversation with Javi about this. Could we leave the loading screen until the custom basemap is loaded?

Reply:
@aabdaab Well, I think it doesn't work that way, because this will set the background color of the loading screen, you're setting the background color of the globe here:
![image](https://user-images.githubusercontent.com/15097138/62288793-0fb12c00-b455-11e9-8926-28024bba39ba.png)
I 'd suggest to remove `style={{ backgroundColor: '#0A212E' }}` from the WebScene and wrap the `loadElement` element into div with dark background color.

Reply:
So @fannycc I think the best solution for us would be to use the Firefly as `basemap` that way we would avoid fetching the tiles for the `satellite imagery` (that we are currently using as `basemap`) and the Firefly when we just want to display the Firefly. That would have two main benefits for us, one would be that we wouldn't see this rendering of the Firefly over the satellite `basemap` and the other one is that we would increase the performance of the `map` as we will only be `fetching` one tile instead of two for the needed globe area (not sure if this performance increase would be perceptible by the user).
But for that we would need to create tiles for the higher zoom levels, and not sure if that's something we can do (maybe @benlaken or @gretacv have something to say here) as has been J.Nelson the one that has been creating the tiles so far.

Reply:
On this image you can see that we are fetching the satellite imagery although we are just _covering_ it with the vibrant map tiles
![image](https://user-images.githubusercontent.com/6906348/62291433-3c207480-b464-11e9-94c0-8a80b13c6a3e.png)

Reply:
whatever works best :)